config/environments/production.rb

```diff
@@ -82,6 +82,15 @@
     config.logger = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  # do not log these huge params in order to prevent wasting disk space
+  config.filter_parameters += [
+    :exercise_calculation_updates,
+    :student_clue_requests,
+    :student_clue_updates,
+    :teacher_clue_updates,
+    :spe_updates,
+    :pe_updates
+  ]
```

Comment:
we need input from @kevinburleigh75  or @Dantemss re: if removing some of these makes the logs useless.